### PR TITLE
Load user on opening website

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**
Implemente ability to get current user

### CODE REVIEW

File: frontend/src/app/components/create-organization/create-organization-dialog/create-organization-dialog.component.html [hunk 1]
```diff
@@ -18,11 +19,12 @@ <h1 mat-dialog-title>Create organization</h1>
             [class]="this.btnClass">
         </tasque-button>
         <tasque-button
-            [mat-dialog-close]="true"
+            [mat-dialog-close]="isSucced"
```
Comment (ismv) [inline frontend/src/app/components/create-organization/create-organization-dialog/create-organization-dialog.component.html]: isSuccessful or isSuccess

Comment (DaniilPoiarkov) [reply frontend/src/app/components/create-organization/create-organization-dialog/create-organization-dialog.component.html]: > isSuccessful or isSuccess

Done

File: backend/Tasque.Core.WebAPI/Controllers/UserController.cs [hunk 2]
```diff
@@ -47,10 +40,21 @@ public async Task<IActionResult> EditPassword([FromBody] PasswordEditDto dto)
...
+
         private int GetCurrentUserId()
         {
-            /* Just a stub. Should be implemented */
-            return 1;
+            var id = User.Claims.FirstOrDefault(x => x.Type == "id")?.Value;
+            if (string.IsNullOrEmpty(id))
+                throw new InvalidTokenException("Invalid access token");
+
+            return int.Parse(id);
```
Comment (Anatolii-Ivanov) [inline backend/Tasque.Core.WebAPI/Controllers/UserController.cs]: Check https://trello.com/c/ImVlCs4P/97-create-services-to-get-userid-from-token for this part

File: frontend/src/app/components/create-organization/create-organization.component.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ export class CreateOrganizationComponent implements OnInit {
   @Input() public currentUser: UserModel = {
     name: 'Login1',
     email: 'testlogin@gmail.com',
-    id: 1,
+    id: 20,
     salt: 'Salt',
     password: 'Password'
   };
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/components/create-organization/create-organization.component.ts]: Can you update this parts to load real user, not mocked?

Comment (Anatolii-Ivanov) [reply frontend/src/app/components/create-organization/create-organization.component.ts]: All such parts

Comment (DaniilPoiarkov) [reply frontend/src/app/components/create-organization/create-organization.component.ts]: Done

File: backend/Tasque.Core.WebAPI/Controllers/UserController.cs (hunk 2 above)
Comment (DaniilPoiarkov) [reply backend/Tasque.Core.WebAPI/Controllers/UserController.cs]: > Check https://trello.com/c/ImVlCs4P/97-create-services-to-get-userid-from-token for this part

Done

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by andpii — APPROVED** [review]

**Pull request merged**